.travis.yml

```diff
@@ -2,16 +2,22 @@ language: python
 python:
   - "2.7"
   - "3.4"
+before_install:
+  - sudo apt-get install jq
+  - wget -O ~/codacy-coverage-reporter-assembly-latest.jar $(curl https://api.github.com/repos/codacy/codacy-coverage-reporter/releases/latest | jq -r .assets[0].browser_download_url)
 install:
     - pip install -r requirements.txt
     - pip install braintree coveralls mock mysqlclient
 env:
   - DB=sqlite
   - DB=mysql
   - DB=postgres
+  - CODACY_PROJECT_TOKEN=0925482d672041289fc7394e23d02255
```

Comment:
<img width="619" alt="screen shot 2017-09-04 at 12 54 42" src="https://user-images.githubusercontent.com/10899666/30021698-ebdadf64-9170-11e7-8f0c-9ce5cb5dc870.png">


Not a good idea to include the project token (and any other sensitive info) in a public file. I recommend that you revoke this token as mentioned in [this](https://goo.gl/65vjpB) article and configure a new token as a travis project environment variable as specified [here](https://goo.gl/pkcLuw)